_blog/2018/v1alpha3-routing.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "Introducing the Istio v1alpha3 routing API"
+overview: Introduction, motivation and design principles for the Istio v1alpha3 routing API. 
+publish_date: April 19, 2018
+subtitle:
+attribution: Frank Budinsky and Shriram Rajagopalan
+
+order: 90
+
+layout: blog
+type: markdown
+redirect_from: "/blog/v1alpha3-routing.html"
+---
+
+{% include home.html %}
+
+Up until now, Istio has provided a simple API for traffic management using four configuration resources:
+`RouteRule`, `DestinationPolicy`, `EgressRule`, and (Kubernetes) `Ingress`.
+With this API, users have been able to easily manage the flow of traffic in an Istio service mesh.
+The API has allowed users to route requests to specific versions of services, inject delays and failures for resilience
+testing, add timeouts and circuit breakers, and more, all without changing the application code itself.
+
+While this functionality has proven to be a very compelling part of Istio, user feedback has also shown that this API does
+have some shortcoming, specifically when using it to manage very large applications containing thousands of services, and
+when working with protocols other than HTTP. Furthermore, the use of Kubernetes `Ingress` resources to configure external
+traffic has proven to be woefully insufficient for our needs.
+
+To address these, and other concerns, a new traffic management API, a.k.a. `v1alpha3`, is being introduced, which will
+completely replace the previous API going forward. Although the `v1alpha3` model is fundamentally the same, it is not
+backward compatible and will require manual conversion from the old API. A
+[conversion tool]({{home}}/docs/reference/commands/istioctl.html#istioctl%20experimental%20convert-networking-config)
+is included in the next few releases of Istio to help with the transition.
+
+To justify this disruption, the `v1alpha3` API has gone through a long and painstaking community
+review process that has hopefully resulted in a greatly improved API that will stand the test of time. In this article,
+we will introduce the new configuration model and attempt to explain some of the motivation and design principles that
+influenced it.
+
+## Design principles
+
+A few key design principles played a role in the routing model redesign:
+
+* Explicitly model infrastructure as well as intent. For example, in addition to configuring an ingress gateway, the
+  component (controller) implementing it can also be specified.
+* The authoring model should be "producer oriented" and "host-centric" as opposed to compositional. For example, all
+  rules associated with a particular host are configured together, instead of individually.
+* Clear separation of routing from post-routing behaviors.
+
```

Comment:
might be good to clarify "post-routing behaviors" -> "post-routing behaviors as defined in destination rules".  It took me a while to realize that is what you are referring to here.